### ISSUE TITLE
[0.10.0-nightly] error reporting for "show tags" can be improved

### ISSUE THREAD

**Issue by makapuf** [influxdata/influxdb#5297]
On command line interface, the error reporting when typing 
```
show tag keys whatever whatever
```
answers with 
```
ERR: error parsing query: found whatever, expected SELECT, DELETE, SHOW, CREATE, DROP, GRANT, REVOKE, ALTER, SET at line 1, char 15
``` 

Where the most useful word would be 'FROM <measurement>', it's not proposed.
(seen in 0.9.4.2)

**Comment by zstyblik** [comment]
Valid for 0.10.0-nightly-3264ab0.

**Comment by jsternberg** [comment]
This was fixed in 0.12 (I think). It should work in 0.13. There was an issue where the semi-colon was optional so it would parse the first section `SHOW TAG KEYS` and then try to parse a second statement where you would then get the error message.
